### PR TITLE
[MRG] Updated urls.

### PR DESCRIPTION
**Pull request by jaeilepp**
Closes https://github.com/mne-tools/mne-python/issues/2981

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -44,8 +44,6 @@ dependencies:
     - python -c "import mne; mne.datasets.testing.data_path(verbose=True);";
     - python -c "import mne; mne.datasets.spm_face.data_path(verbose=True);";
     - python -c "import mne; mne.datasets.somato.data_path(verbose=True);";
-    - python -c "import mne; mne.datasets.brainstorm.bst_auditory.data_path(verbose=True)" --accept-brainstorm-license;
-    - python -c "import mne; mne.datasets.brainstorm.bst_resting.data_path(verbose=True);" --accept-brainstorm-license;
```
Comment (agramfort) [inline circle.yml]: why did you remove this?

Comment (Eric89GXL) [inline circle.yml]: aren't used in examples yet, save bandwidth

**Comment by Eric89GXL** [comment]
Did you test the code? I suspect there will be some MD5sums that need updating.

**Comment by jaeilepp** [comment]
Yeah I tested it. The brainstorm data doesn't seem to have a hash.

**Comment by agramfort** [comment]
is it the only dataset not having one?

**Comment by jaeilepp** [comment]
Yep.

**Comment by agramfort** [comment]
that's good enough for me.

@Eric89GXL if you feel like a hash would be nice to have. I'll let you take care of it.
